data-prepper-plugins/peer-forwarder/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1,2 @@
+# https://github.com/mockito/mockito/wiki/What%27s-new-in-Mockito-2#mock-the-unmockable-opt-in-mocking-of-final-classesmethods
+mock-maker-inline
```

Comment:
QUES: what class methods is this for?

Reply:
Allows mocking of final class methods with vanilla Mockito (I'll add another comment line).

Had to bring it in to mock Armeria's "DnsAddressEndpointGroup" class, otherwise the unit test would actually try making a network call for the DNS lookup.